### DISCUSSION TITLE
Content API filtering assets with multiple values

### DISCUSSION THREAD

**Discussion by ewhicher** [statamic/cms#5759] [Troubleshooting] [answered]
I have a taxonomy terms field on all my assets with multi select enabled.
Should something like this work with the Content API? It doesn't return any results, even tho the tags are there in the data...

`/api/assets/container-name?filter[tags:contains]=white|green`

Is there a better way?

**Comment by Eworm** [comment]
I'm stuck wondering the same thing. It seems strange that filtering with an array isn't possible.

**Comment by joshuablum** [comment, marked as answer]
Hey @ewhicher and @Eworm,

This was added in Statamic [v3.3.34](https://github.com/statamic/cms/releases/tag/v3.3.34) and recently got documented by @tomhelmer.

Have a look at [the docs on it](https://statamic.dev/rest-api#taxonomies).

You can now do something like: `/endpoint?filter[taxonomy:{handle}]={value},{value},{value}`

Hope that helps :)

**Comment by Eworm** [comment]
Nice, thanks for the comment!!

**Comment by ewhicher** [comment]
Ah amazing, thanks for the heads up @joshuablum 🙌
